resources/icons/16px/range-ring.svg

```diff
@@ -0,0 +1,10 @@
+</g>
+<defs>
+<clipPath id="clip0_9748_11">
+<rect width="16" height="16" fill="white"/>
+</clipPath>
+</defs>
+</svg>
```

Comment:
I think we can clean up some of the SVG markup here. The `<clipPath>`, `<rect>`, group, and fill/clip rules all seem unnecessary. Additionally, I think we can remove all fills and let the path inherit the default fill color.
```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
  <path d="M8 0C7.64677 0 7.29839 0.0229549 6.95639 0.0675665C6.40874 0.139002 6.0227 0.640866 6.09414 1.18851C6.16557 1.73616 6.66744 2.1222 7.21508 2.05077C7.47153 2.01731 7.73349 2 8 2C8.26651 2 8.52847 2.01731 8.78492 2.05077C9.33256 2.1222 9.83443 1.73616 9.90586 1.18851C9.9773 0.640866 9.59126 0.139002 9.04361 0.0675663C8.70161 0.0229549 8.35323 0 8 0ZM4.34784 3.23912C4.78576 2.9026 4.86795 2.27479 4.53143 1.83688C4.19491 1.39896 3.5671 1.31676 3.12918 1.65328C2.57534 2.07889 2.07889 2.57534 1.65328 3.12918C1.31676 3.5671 1.39896 4.19491 1.83688 4.53143C2.27479 4.86795 2.9026 4.78576 3.23912 4.34784C3.55881 3.93183 3.93183 3.55881 4.34784 3.23912ZM12.8708 1.65328C12.4329 1.31676 11.8051 1.39896 11.4686 1.83688C11.132 2.27479 11.2142 2.9026 11.6522 3.23912C12.0682 3.55881 12.4412 3.93183 12.7609 4.34784C13.0974 4.78576 13.7252 4.86795 14.1631 4.53143C14.601 4.19491 14.6832 3.5671 14.3467 3.12918C13.9211 2.57534 13.4247 2.07889 12.8708 1.65328ZM15.9324 6.95639C15.861 6.40874 15.3591 6.0227 14.8115 6.09414C14.2638 6.16557 13.8778 6.66744 13.9492 7.21508C13.9827 7.47153 14 7.73349 14 8C14 8.26651 13.9827 8.52847 13.9492 8.78492C13.8778 9.33256 14.2638 9.83443 14.8115 9.90586C15.3591 9.9773 15.861 9.59126 15.9324 9.04361C15.977 8.70161 16 8.35323 16 8C16 7.64677 15.977 7.29839 15.9324 6.95639ZM2.05077 7.21508C2.1222 6.66744 1.73616 6.16557 1.18851 6.09414C0.640866 6.0227 0.139002 6.40875 0.0675663 6.95639C0.0229549 7.29839 0 7.64677 0 8C0 8.35323 0.0229549 8.70161 0.0675665 9.04361C0.139002 9.59126 0.640866 9.9773 1.18851 9.90586C1.73616 9.83443 2.1222 9.33256 2.05077 8.78492C2.01731 8.52847 2 8.26651 2 8C2 7.73349 2.01731 7.47153 2.05077 7.21508ZM3.23912 11.6522C2.9026 11.2142 2.27479 11.132 1.83688 11.4686C1.39896 11.8051 1.31676 12.4329 1.65328 12.8708C2.07889 13.4247 2.57534 13.9211 3.12918 14.3467C3.5671 14.6832 4.19491 14.601 4.53143 14.1631C4.86795 13.7252 4.78576 13.0974 4.34784 12.7609C3.93183 12.4412 3.55881 12.0682 3.23912 11.6522ZM14.3467 12.8708C14.6832 12.4329 14.601 11.8051 14.1631 11.4686C13.7252 11.132 13.0974 11.2142 12.7609 11.6522C12.4412 12.0682 12.0682 12.4412 11.6522 12.7609C11.2142 13.0974 11.132 13.7252 11.4686 14.1631C11.8051 14.601 12.4329 14.6832 12.8708 14.3467C13.4247 13.9211 13.9211 13.4247 14.3467 12.8708ZM9.04361 15.9324C9.59126 15.861 9.9773 15.3591 9.90586 14.8115C9.83443 14.2638 9.33256 13.8778 8.78492 13.9492C8.52847 13.9827 8.26651 14 8 14C7.73349 14 7.47153 13.9827 7.21508 13.9492C6.66744 13.8778 6.16557 14.2638 6.09414 14.8115C6.0227 15.3591 6.40875 15.861 6.95639 15.9324C7.29839 15.977 7.64677 16 8 16C8.35323 16 8.70161 15.977 9.04361 15.9324ZM8 6C9.10457 6 10 6.89543 10 8C10 9.10457 9.10457 10 8 10C6.89543 10 6 9.10457 6 8C6 6.89543 6.89543 6 8 6ZM12 8C12 5.79086 10.2091 4 8 4C5.79086 4 4 5.79086 4 8C4 10.2091 5.79086 12 8 12C10.2091 12 12 10.2091 12 8Z" />
</svg>

```